cli/types/index.d.ts

```diff
@@ -4,7 +4,7 @@
 //                 Mike Woudenberg <https://github.com/mikewoudenberg>
 //                 Robbert van Markus <https://github.com/rvanmarkus>
 //                 Nicholas Boll <https://github.com/nicholasboll>
-// TypeScript Version: 2.9
+// TypeScript Version: 3.4

```

Comment:
this is a pretty big jump. Is this normal? Should we maintain 2 versions of type definitions: one v2 and another v3?